### PR TITLE
[Feature][Merv] Add residential properties page

### PR DESCRIPTION
**Pull request by jdesolate**
- Add initial residential properties page

### CODE REVIEW

File: pages/dev/index.tsx [hunk 1]
```diff
@@ -109,7 +109,12 @@ function DevHandout() {
...
         }
-        values={["100", "200", "300", "400"]}
+        values={{
+          amount: "400",
+          area: "100",
+          car: "200",
+          people: "300",
+        }}
       />
     </>
```
Comment (tapsTapayan) [inline pages/dev/index.tsx]: Move data to pseudodata.ts

File: pagination/ResidentialProperties/index.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    },
+    {
+      description:
+        "Home of Violet people.",
+      imageSrc: ResidentialImage,
+      title: "Violet Home",
+      type: "Residential",
+      values: values,
+    },
+  ];
```
Comment (tapsTapayan) [inline pagination/ResidentialProperties/index.tsx]: move this data to pseudodata.ts

**Review by tapsTapayan — CHANGES_REQUESTED** [review]
Move lang ang data sa pseudodata

File: pages/dev/index.tsx (hunk 1 above)
Comment (jdesolate) [reply pages/dev/index.tsx]: sa kani nash ba need pani nato? like ang dev page?

Comment (jdesolate) [reply pages/dev/index.tsx]: resolved

File: pagination/ResidentialProperties/index.tsx (hunk 2 above)
Comment (jdesolate) [reply pagination/ResidentialProperties/index.tsx]: resolved

File: pagination/ResidentialProperties/index.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  useEffect(() => {
+    async function getResidentialProperties() {
+      setIsLoading(true);
+      const data = residential_properties_data;
+
+      if (data) {
+        setResidentialProperties(data);
+      }
+      
+      setTimeout(()=>setIsLoading(false), 1000);
```
Comment (jdesolate) [inline pagination/ResidentialProperties/index.tsx:36]: Don't mind this lng haha iremove raning setTimeout if naa njud tong await fetching sa data or idk unsay way ani sa kang jez hahaha para rni mu loading

**Review by CyberBlaze007 — APPROVED** [review]
- @jdesolate approved to unblock this pr

**Pull request merged**